learntools/ml-insights/nbs/tut3.ipynb

```diff
@@ -0,0 +1,215 @@
+      "text/plain": [
+       "<Figure size 1080x684 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from matplotlib import pyplot as plt\n",
+    "from pdpbox import pdp, get_dataset, info_plots\n",
+    "\n",
+    "# Create the data that we will plot\n",
+    "pdp_goals = pdp.pdp_isolate(model=my_model, dataset=val_X, model_features=feature_names, feature='Goal Scored')\n",
+    "\n",
+    "# plot it\n",
+    "pdp.pdp_plot(pdp_goals, 'Goal Scored')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Scoring a goal substantially increases your chances of winning \"Player of The Game.\"  But extra goals beyond that appear to have little impact on predictions.\n",
```

Comment:
I'm curious why the points in these plots are unevenly spaced along the x axis.

Reply:
Also:
- in this case of the first plot, why do we max out at 3 goals scored? Is that the largest number of goals scored by any team in any game in the dataset? Seems low.
- Even when goals scored is maxed at 3, it seems like the average P(man of the game) is only 0.25. Shouldn't the global average be 0.5? Or does not every game have a "man of the game"?
- What's the blue stuff? Interquartile range?